src/components/Weather.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+
+const WeatherComponent = () => {
+  const [weatherData, setWeatherData] = useState(null);
+
+  useEffect(() => {
+    const API_KEY = 'b71ca2424f7ca2f981190ce292e41ccd';
+    const city = 'Tel Aviv';
+    const apiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}&units=metric`;
+
+    fetch(apiUrl)
+      .then(response => response.json())
+      .then(data => {
+        setWeatherData(data);
+      })
+      .catch(error => console.error('Error fetching data:', error));
```

Comment:
you need to make a file for api requests like it says in the conventions,
also, its more recommanded to use axios over fetch